src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -1876,6 +1879,14 @@ private static String handleKeyword(final String field) {
         return field;
     }
 
+    public static DiscoveryNode getLocalNode() {
+        return localNode;
+    }
+
+    public static void setLocalNode(DiscoveryNode node) {
```

Comment:
need this for security Interceptor tests

Reply:
Is it possible to add `localNode` to `SecurityInterceptor` constructor and remove this static method only needed for testing?